src/core/dbus-cgroup.c

```diff
@@ -1950,6 +1951,32 @@ int bus_cgroup_set_property(
                 return 1;
         }
 
+        if (streq(name, "BindNetworkInterface")) {
+                const char *s;
+
+                r = sd_bus_message_read(message, "s", &s);
+                if (r < 0)
+                        return r;
+
+                if (!ifname_valid_full(s, IFNAME_VALID_ALTERNATIVE)) {
+                        return sd_bus_error_setf(reterr_error, SD_BUS_ERROR_INVALID_ARGS, "Invalid interface name: %s", s);;
+                }
```

Comment:
no {} around single line if blocks in our codebase please.

there's a double `;;` at the end here.